ert/ensemble_evaluator/activerange.py

```diff
@@ -0,0 +1,113 @@
+"""This module contains functionality of representing a list of booleans as
```

Comment:
Will we have two implementations of the same thing, considering `RangeStringArgument` exists?

Reply:
Thanks, that class was yet undiscovered ;)

Reply:
What do you mean? Should we have two `mask_to_rangestring` in our code base? https://github.com/equinor/ert/blob/007f0077b6cfd7bcc3d7e82f29904038cbe69a20/ert_gui/ertwidgets/models/activerealizationsmodel.py

Reply:
Left as a pending task in https://github.com/equinor/ert/issues/3121